app/grandchallenge/hanging_protocols/forms.py

```diff
@@ -116,3 +129,86 @@ def _validate_slice_plane_indicator(self, *, viewport, viewport_names):
                 error=f"Viewport {viewport['viewport_name']} has a slice_plane_indicator that is the same as the viewport_name.",
                 field="json",
             )
+
+
+class ViewContentExampleMixin:
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        if self.instance:
+            interface_slugs = self.instance.interfaces.values_list(
+                "slug", flat=True
+            )
+
+            if interface_slugs.count() > 0:
+                self.fields[
+                    "view_content"
+                ].help_text += f"The following interfaces are used in your {self.instance._meta.verbose_name}: {oxford_comma(interface_slugs)}. "
+
+            view_content_example = self.generate_view_content_example()
+
+            if view_content_example:
+                self.fields[
+                    "view_content"
+                ].help_text += f"Example usage: {view_content_example}. "
+            else:
+                self.fields[
+                    "view_content"
+                ].help_text += "No interfaces of type (image, chart, pdf, mp4, thumbnail_jpg, thumbnail_png) are used. At least one interface of those types is needed to config the viewer. "
+
+        self.fields["view_content"].help_text += format_lazy(
+            'Refer to the <a href="{}">documentation</a> for more information',
+            reverse("documentation:detail", args=["viewer-content"]),
+        )
+
+    def generate_view_content_example(self):
+        images = list(
+            self.instance.interfaces.filter(
+                kind=InterfaceKindChoices.IMAGE
+            ).values_list("slug", flat=True)
+        )
+        mandatory_isolation_interfaces = list(
+            self.instance.interfaces.filter(
+                kind__in=InterfaceKind.interface_type_mandatory_isolation()
+            ).values_list("slug", flat=True)
+        )
+
+        if not images and not mandatory_isolation_interfaces:
+            return None
+
+        overlays = list(
+            self.instance.interfaces.exclude(
+                kind__in=InterfaceKind.interface_type_undisplayable()
+                | InterfaceKind.interface_type_mandatory_isolation()
+                | {InterfaceKindChoices.IMAGE}
+            ).values_list("slug", flat=True)
+        )

```

Comment:
See my comment above. 

And also: does this really only return overlays? What about strings, integers etc., should those be considered overlays and are they displayable?  I guess it's fine, or else they would be listed in `interface_type_undisplayable`, but just wondering.

Reply:
According the discussion on the pitch, strings, bools and integers can be used in combination with Images and shown as overlays in the viewport.

from Miriam

> because for example you can add CI's with text, which will be shown as labels on the viewport

from James

> Also can't some non-image interfaces also be assigned to view items: pdf files and charts in isolation, floats and bools in combination with images, etc?

list comprehension point is taken.
Done.